lib/port_scanner.py

```diff
@@ -1,44 +1,50 @@
 import socket
 import threading
-from queue import Queue
 from lib.utils import Color
 
-queue = Queue()
 
-
-def scan_port(host, port):
+def scan_port(host, port, results):
```

Comment:
I think think this function should return a `bool` and the `scan_thread` function can check if the result was `True`, and if so it would append the results list.

Reply:
Done